save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/vulnerability/VulnerabilityDto.kt

```diff
@@ -68,5 +73,9 @@ data class VulnerabilityDto(
             emptyList(),
             VulnerabilityStatus.CREATED,
         )
+        val vulnerabilityPrefixes = listOf(
```

Comment:
it's not practical that prefixes are in code -- adding a new one requires a build and release. Maybe move it to configuration at least?

Reply:
Will do it later, when the list is completed - work is still in progress so it will be recompiled anywhay